<ui version="4.0">
  <class>SettingDialog</class>
  <widget class="QDialog" name="SettingDialog">
    <property name="geometry">
      <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>200</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Chat Settings</string>
    </property>
    <layout class="QVBoxLayout">
      <property name="spacing">
        <number>6</number>
      </property>
      <item>
        <layout class="QHBoxLayout">
          <property name="spacing">
            <number>6</number>
          </property>
          <!-- first group -->
          <item>
            <layout class="QVBoxLayout">
              <property name="spacing">
                <number>6</number>
              </property>
              <item>
                <widget class="QLabel">
                  <property name="text">
                    <string>Nickname</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel">
                  <property name="text">
                    <string>Chatroom</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel">
                  <property name="text">
                    <string>Prefix</string>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <!-- first group -->

          <!-- second group -->
          <item>
            <layout class="QVBoxLayout">
              <property name="spacing">
                <number>6</number>
              </property>
              <item>
                <widget class="QLineEdit" name="nickEdit" />
              </item>
              <item>
                <widget class="QLineEdit" name="roomEdit" />
              </item>
              <item>
                <widget class="QLineEdit" name="prefixEdit" />
              </item>
            </layout>
          </item>
          <!-- second group -->
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <widget class="QPushButton" name="cancelButton">
              <property name="text">
                <string>Cancel</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="okButton">
              <property name="text">
                <string>OK</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>
